randip/main.go

```diff
@@ -0,0 +1,152 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/fatih/color"
+	"github.com/ipinfo/cli/lib"
+	"github.com/ipinfo/cli/lib/complete/install"
+	"github.com/spf13/pflag"
+)
+
+var progBase = filepath.Base(os.Args[0])
+var version = "1.0.0"
+
+func printHelp() {
+	fmt.Printf(
+		`Usage: %s [<opts>] 
+
+Description:
+  Generates random IP/IPs.
```

Comment:
Please update this one too.

Reply:
Can you please elaborate a bit more on what you want me to update in this file?

Reply:
Same as above - should be "Generate random IPs".